### PR TITLE
DOC: Add gallery example of polygon plotting with folium

### PR DESCRIPTION
**Pull request by donlo**
Attempt to address #529

### CODE REVIEW

File: examples/plotting_with_folium.ipynb [hunk 1]
```diff
@@ -0,0 +1,519 @@
...
+   "source": [
+    "df.plot(figsize=(6, 6))\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Project geometry to WGS84 (EPSG4326)"
```
Comment (martinfleis) [inline examples/plotting_with_folium.ipynb]: Can you explain in the text why are you reprojecting it?

Comment (donlo) [reply examples/plotting_with_folium.ipynb]: added explanation!

**Comment by donlo** [comment]
Thanks for the review! I have added the explanation of the re-projection and modified the titles and a bit of contents.

**Comment by martinfleis** [comment]
Thank you for such a quick response. Just one final thing. Which version of geopandas are you using? Because when you call `print(df.crs)` you get `{'proj': 'lcc', 'lat_1': 40.66666666666666, 'lat_2': 41.03333333333333, 'lat_0': 40.16666666666666, 'lon_0': -74, 'x_0': 300000, 'y_0': 0, 'datum': 'NAD83', 'units': 'us-ft', 'no_defs': True}` while I get `{'init': 'epsg:2263'}` (on 0.5.0). Everything else is the same.

**Comment by donlo** [comment]
Oh I used 0.4.1 for this example. Thanks for checking. Mine outputs `{'init': 'epsg:2263'}` too now after updating to 0.5.0

**Comment by martinfleis** [comment]
Thanks for this work!
Merging now as the example is nice and ready. However, we have to find a proper way how to embed it in docs (#280), which I aim to work on this week.

**Comment by donlo** [comment]
Again thanks for the review!

**Pull request merged**